.github/workflows/go.yml

```
@@ -53,29 +53,31 @@
+        uses: dorny/paths-filter@v2
         with:
-          files: |
-            **/*.sol
+          filters: |
+            sol:
+              - '**/*.sol'
 
       - name: Check For GolangCI Changes
         id: golangci_changed
-        uses: tj-actions/changed-files@v26.1
+        uses: dorny/paths-filter@v2
```

actionlint (1.7.4)

the runner of "dorny/paths-filter@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)